src/intl/index.js

```diff
@@ -156,12 +156,13 @@ const getUrl = ({ getState, langPackLocale, componentKey }) => {
   ].join('/')}.json`;
 };
 
-const fetchLanguagePack = ({
+const fetchLanguagePack = async ({
```

Comment:
Why does this need to be async now? it doesn't use await, and always returns a promise anyway?

If it is to be marked as async, can the returns all be changed to literal returns and throws?

Reply:
I had started to refactor this function to async/await then decided not to. This is left over from that. Will clean up